package.json

```diff
@@ -58,10 +58,14 @@
     "query-string": "^5.0.0"
```

Comment:
It sounds like we should add @babel/runtime as a dependency from the [docs](https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers).  Not sure if you noticed that or tried and had an issue?

Reply:
Yeah, really odd that I haven't run into any issues with this... I'll dive into it some more and see what's going on.

Reply:
Sounds good.  it's not clear on two fronts:

* is this a dep or devDep?  it's used at dev/build time, but it is also a runtime dep
* it says to "tell rollup to treat it as you external dependency when bundling for cjs & es"

Reply:
I've been digging into this this evening a bit as well.  The benefit of the runtime option is to both reduce the babel helper duplication and to provide a namespace in which to apply polyfills so you don't pollute the global namespace for downstream consumers.  Which sounds perfect, but ...

the rubs seem to be:
* only works in esm and cjs formats [ref](https://github.com/rollup/plugins/tree/master/packages/babel#injected-helpers)
* "The transformation plugin is typically used only in development" [ref](https://babeljs.io/docs/en/babel-plugin-transform-runtime#installation)

So, it seems we would potentially have a different set of babel plugins and rollup config for umd vs cjs/esm bundles.  And, potentially, another config for our dev server that's using the transform vs the external @babel/runtime dependency.

I tried adding @babel/runtime as a dep and specifying it as an external resource in rollup config but it didn't modify the output.  Presumably because we're already using inline everywhere as it's "hard-coded" in .babelrc.

So, tl;dr, seems like we've got more research to do to get all these packaged in the way we want them.  It looks like you can use getBabelOutputPlugin to control the babel plugins used (and presumably only have very common babel bits in babel.rc config?)  But, I don't love the idea of having every artifact use different babel config.  It may be necessary, but seems like a PITA.

Let's dig into this a bit tomorrow.